styles.go

```diff
@@ -642,17 +642,224 @@ var (
 		HTMLSpan:  ansi.StyleBlock{},
 	}
 
+	// TokyoNightStyle is the tokyo night style.
```

Comment:
Should we move `dracula.go` here as well?

Reply:
Ah I see what you mean. I think so. We can re-organize them another time if this file gets too big, but if all of our themes are there except Dracula, then yeah def worth moving it to that file. I'll move it over :)

Reply:
I can see `styles.go` growing so large that way. Perhaps we could move `dracula.go` to `style_dracula.go`, wdyt?

Reply:
Related: https://github.com/charmbracelet/glamour/issues/298

Reply:
kk I'll play with it a bit to see how we can clean it up better